doc/appendix-de-kasssichv/appendix-de-kasssichv.md

```diff
@@ -96,57 +96,112 @@ For Germany (DE) the country code is `0x4445`. Thus, the value for an unknown ft
 | **Value** | **Description** | **Service- Version** |
 |----------------------|-----------------------------------------------------------------------------------------------------------------------------------|---------------------|
 | `0x4445000000000000` | "default value"<br />unknown payment type: automatic processing through the fiskaltrust.SecurityMechanisms settings is attempted. | 1.3-                |
-| `0x4445000000000000` | "unknown payment type for DE"<br />This is handled like a Start Receipt.                                                          | 1.3-                |
+| `0x4445000000000000` | "unknown payment type for DE"<br />This is handled like a pos-receipt.                                                          | 1.3-                |
+| `0x444500000000xxxx` | start-transaction<br />starts a new, unfinised action. use ChargeItems and PayItems to hand over already known details for final receipt. using the same cbTerminalID and cbReceiptReferece in further calls connects the action items. <br />this works only on explicit flow. calling with the ReceiptCaseFlag 0xyyyy ends up in an exception. | 1.3- |
+| `0x444500000000xxxx` | update-transaction<br />updates an ongoing action. use ChargeItems and PayItems to hand over all details for final receipt. using the same cbTerminalID and cbReceiptReferece in further calls connects the action items. <br />this works only on explicit flow. calling with the ReceiptCaseFlag 0xyyyy ends up in an exception. | 1.3- |
+| `0x444500000000xxxx` | dela-transaction<br />updates an ongoing action. use ChargeItems and PayItems to hand changes for final receipt. using the same cbTerminalID and cbReceiptReferece in further calls connects the action items. <br />this works only on explicit flow. calling with the ReceiptCaseFlag 0xyyyy ends up in an exception. | 1.3- |
```

Comment:
This ist "delta" or "increment". 
These are the new data created within 45 seconds of the start of a data update (update transaction); (e.g. reordering drinks in a restaurant).

Reply:
For me the start and probably the fail-transaction are fine in the ftReceiptCase table.
But shouldn't the update and delta transactions be content of the ftReceiptCaseFlags? So that I can decide, to do an update specifying in the ReceiptCase if it "probably" will be an invoice, delivery note or whatever it is?

Is it on purpose, that the exact numbers are not defined now and xxxx is written in all lines? As there are already different systems in AT and FR, the numbering could be consecutive anyway?

Reply:
update-transaction and delta-transaction are separated cases. in the background they do an UpdateTransaction on the TSE side.
there are two concepts explained in the beginning. in update you hand over the current state over all. in delta you hand over only the changes since last call on start-transaction or delta-transaction.
because of different TSE allow only the one or only the other flow, we will allow both of them and shrink down the data to proper hand them over to the used TSE.